liberty-maven-plugin/src/it/springboot-3-tests/springboot-3-appsdirectory-apps-it/src/test/java/application/InstallSpringBoot30AppIT.java

```diff
@@ -27,6 +27,9 @@ public void testThinApplicationExistsInAppsDirectory() throws Exception {
 
         File f = new File("target/liberty/wlp/usr/servers/test/apps/thin-springboot-3-appsdirectory-apps-it-1.0.0.Final-exec.jar");
         assertTrue(f.getCanonicalFile() + " doesn't exist. Plugin failed to place the file at right destination.", f.exists());
+        File f2 = new File("target/liberty/wlp/usr/servers/test/configDropins/defaults");
+        assertTrue(f2.getCanonicalFile() + " folder doesn't exist. Plugin failed to create the config dropins folder ", f2.exists());
+        assertTrue(f2.getCanonicalFile() + " folder doesn't contain any files. Plugin failed to place config xml at right destination.", f2.list().length == 1);
```

Comment:
Can we verify the contents of the generated file?

Reply:
Added